lib/Net/SAML2/Protocol/Assertion.pm

```diff
@@ -296,4 +309,10 @@ sub valid {
     return 1;
 }
 
+sub success {
```

Comment:
Also should have some pod here to describe success and point them to valid for full validation of the assertion